### PR TITLE
bug 1150118 - A bit of JS cleanup

### PR DESCRIPTION
**Pull request by darkwing**

### CODE REVIEW

File: media/js/fonts.js [hunk 1]
```diff
@@ -111,12 +111,12 @@
             var fontAttribute = index;
             var fontLoaded = true;
             $.each(this, function() {
-                if(this == false) {
+                if(!this) {
```
Comment (darkwing) [inline media/js/fonts.js]: I wonder if we just do `fontLoaded = this;`

Comment (stephaniehobson) [inline media/js/fonts.js]: Right now it loops through each font and checks if it is loaded, changing `fontLoaded` to false if it hasn't. If we just do `fontLoaded = this` then `fontLoaded` will have the value of the last font in the loop, won't it? Or am I misunderstanding where we'd insert it?

File: media/js/fonts.js [hunk 2]
```diff
@@ -111,12 +111,12 @@
...
                     fontLoaded = false;
                 }
             })
-            if(fontLoaded == true) {
-                $('html').attr("data-"+fontAttribute, true);
+            if(fontLoaded) {
+                $('html').attr('data-' + fontAttribute, true);
```
Comment (stephaniehobson) [inline media/js/fonts.js]: At least I'm internally consistent with my incorrect use of quotes. >_<

File: media/js/fonts.js (hunk 1 above)
Comment (stephaniehobson) [inline media/js/fonts.js]: Can we make the same replacement on lines 83 and 87?

**Comment by stephaniehobson** [comment]
Tested, and it works, so we could merge and have better code :) Or we could make additional changes, mostly those suggested by jshint.

**Comment by darkwing** [comment]
Updated!

**Comment by stephaniehobson** [comment]
R+ once Travis gives the okay.

**Comment by darkwing** [comment]
Kicked!

File: media/js/fonts.js (hunk 2 above)
Comment (kleberveridiano) [inline media/js/fonts.js]: to me it´s fine!

**Comment by kleberveridiano** [comment]
Great!

**Pull request merged**